src/multi-select/dropdown.tsx

```diff
@@ -101,11 +101,14 @@ const Dropdown = () => {
     isInternalExpand && setExpanded(isLoading || disabled ? false : !expanded);
   };
 
-  const handleClearSelected = (e) => {
-    e.stopPropagation();
-    onChange([]);
-    isInternalExpand && setExpanded(false);
-  };
+  const handleClearSelected = useCallback(
+    (e) => {
+      e.stopPropagation();
+      onChange([]);
+      isInternalExpand && setExpanded(false);
+    },
+    [onChange, isInternalExpand]
+  );
 
```

Comment:
This will avoid creating a new function on every render, which can improve performance